.github/actions/install-node/action.yml

```diff
@@ -0,0 +1,20 @@
+name: Rake installer  
+jobs:
+description: 'Node installer'
+runs:
+  using: "composite"  
+  steps:                      
+    - name: Install packages
+      run: |
+        sudo apt-get update
+        sudo apt-get -y install libmagickwand-dev imagemagick ghostscript pdftk enscript          
```

Comment:
Is there a specific reason this is inside the "install node" action? I wouldn't expect that these are required for node, but I would expect they're required more broadly, even by things that don't require node. It probably makes sense to have a "install apt packages" action

Reply:
sounds good!

I think it was laziness and a bit lack of understanding the difference between these packages and the connection with node